### PR TITLE
CHANGELOG for 1.6.0

### PR DESCRIPTION
**Pull request by xorye**
Signed-off-by: David Kwon <dakwon@redhat.com>

### CODE REVIEW

**Comment by fbricon** [comment]
need to add https://github.com/redhat-developer/vscode-quarkus/issues/254

**Comment by xorye** [comment]
@fbricon updated

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,30 @@
 # Quarkus Tools for Visual Studio Code Changelog
 
+## 1.6.0 (July 9, 2020)
+
+### Enhancements
+
+ * Add welcome page icon. See [#260](https://github.com/redhat-developer/vscode-quarkus/issues/260).
+ * Update Java version on the welcome page. See [#258](https://github.com/redhat-developer/vscode-quarkus/issues/258).
+ * Quarkus datasource snippet creates an invalid prop. for 1.5. See [#255](https://github.com/redhat-developer/vscode-quarkus/issues/255).
```
Comment (fbricon) [inline CHANGELOG.md]: that should be a bug fix

Comment (xorye) [reply CHANGELOG.md]: updated

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,30 @@
...
+ * Update Java version on the welcome page. See [#258](https://github.com/redhat-developer/vscode-quarkus/issues/258).
+ * Mitigate vscode-quarkus & vscode-spring-boot competing for application.properties. See [#254](https://github.com/redhat-developer/vscode-quarkus/issues/254).
+ * Display project name when adding extensions. See [#215](https://github.com/redhat-developer/vscode-quarkus/issues/215).
+
+### Bug fixes
+
+ * Fix welcome page typos. See [#259](https://github.com/redhat-developer/vscode-quarkus/pull/259).
+ * Quarkus datasource snippet creates an invalid prop. for 1.5. See [#255](https://github.com/redhat-developer/vscode-quarkus/issues/255).
+ * vscode-quarkus should not be a Java formatter. See [#166](https://github.com/redhat-developer/vscode-quarkus/issues/166).
+ * Quarkus datasource snippet is invalid. See [LS#316](https://github.com/redhat-developer/quarkus-ls/issues/316).
```
Comment (fbricon) [inline CHANGELOG.md]: duplicates with #255 (2 lines above)

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,30 @@
...
+ * Missing support for container-image properties. See [LS#315](https://github.com/redhat-developer/quarkus-ls/issues/315).
+ * Filter properties snippet with properties instead of dependency. See [LS#312](https://github.com/redhat-developer/quarkus-ls/issues/312).
+
+### Build
+
+ * Publish releases to open-vsx.org. See [#278](https://github.com/redhat-developer/vscode-quarkus/pull/278).
+ * Migrate vscode-extension-tester to 3.x. See [#266](https://github.com/redhat-developer/vscode-quarkus/issues/266).
```
Comment (fbricon) [inline CHANGELOG.md:24]: missing https://github.com/redhat-developer/vscode-quarkus/issues/267

**Pull request merged**